pkg/zeek/double.go

```diff
@@ -121,12 +121,12 @@ func (d Double) Comparison(op string) (Predicate, error) {
 		case *TypeOfTime:
 			ts, err := DecodeTime(val)
 			if err == nil {
-				return compare(float64(ts), pattern)
+				return compare(float64(ts)/1e9, pattern)
```

Comment:
We cool with the loss of precision here? I don't think it's a big deal, but perhaps @mccanne does. Could use big.Float if this is a concern...

Reply:
zql doubles are specified as float64 so the loss of precision is inherent here unless I'm missing something. 

I think that this is arguing for a syntax for time literals, since time doesn't have precision issues.